cmd/cloudx/relationtuples/relationtuples.go

```diff
@@ -0,0 +1,126 @@
+package relationtuples
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	ketoClient "github.com/ory/keto/cmd/client"
+	"github.com/ory/keto/cmd/relationtuple"
+	"github.com/ory/x/randx"
+
+	"github.com/ory/cli/cmd/cloudx/client"
+)
+
+const FlagAll = "all"
+
+var ErrDeleteMissingAllFlag = fmt.Errorf("please select the tuples to delete or use `--all` to delete all tuples")
+
+func List() *cobra.Command {
+	cmd := relationtuple.NewGetCmd()
+	cmd.Short = "List relation tuples"
+	cmd.Long = `List relation tuples matching the given partial tuple.
+Returns paginated results.`
+	wrapForOryCLI(cmd)
+
+	return cmd
+}
+
+func Delete() *cobra.Command {
+	cmd := relationtuple.NewDeleteAllCmd()
+	wrapForOryCLI(cmd)
+
+	cmd.Flags().Bool(FlagAll, false, "Delete all relation tuples")
+	wrapForDelete(cmd)
+
+	return cmd
+}
+
+func Create() *cobra.Command {
+	cmd := relationtuple.NewCreateCmd()
+	wrapForOryCLI(cmd)
+
+	return cmd
+}
+
+func Parse() *cobra.Command {
+	cmd := relationtuple.NewParseCmd()
+	wrapForOryCLI(cmd)
+
+	return cmd
+}
+
+func forwardConnectionInfo(cmd *cobra.Command) {
+	originalRunE := cmd.RunE
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		_, _, project, err := client.Client(cmd)
+		if err != nil {
+			return err
+		}
+
+		h, err := client.NewCommandHelper(cmd)
+		if err != nil {
+			return err
+		}
+
+		key, err := h.CreateAPIKey(project.Slug, "keto-temp-"+randx.MustString(8, randx.AlphaNum))
+		if err != nil {
+			return err
+		}
+		defer func() { _ = h.DeleteAPIKey(project.Slug, key.Id) }()
```

Comment:
Yeah, I don't know what we should do if that fails. I am already discussing with @zepatrik on how to use the session ID directly in Keto, so that we don't have to do this weird dance of get-a-token, use-a-token, delete-a-token.

But if deletion fails, I guess the key just stays for the time being. It's hard to recover that error.

Reply:
Tracking issue: https://github.com/ory-corp/cloud/issues/3169